src/view/states/ShowState.cpp

```diff
@@ -4,20 +4,24 @@
 
 #include "ShowState.h"
 #include "utilities/TaskConvertors.h"
+#include "utilities/TaskComparators.h"
 
-void ShowState::PrintTasks(const std::vector<RelationalTask>& tasks) {
+void ShowState::PrintTaskWithChildren(const std::vector<RelationalTask>& tasks, const RelationalTask& task_to_print, const std::string& prefix = "") const {
```

Comment:
The default parameter here can cause different behavior between different translation units. Maybe it is better to pass parameter explicitly in call.

Reply:
operativno